core/ui/mpc/fast/WaitForServerStep.tsx

```
@@ -1,5 +1,6 @@
+import { KeygenType } from '@core/mpc/keygen/KeygenType'
```

ESLint

[error] 'KeygenType' is defined but never used.

(@typescript-eslint/no-unused-vars)